storage/src/cache/fscache/mod.rs

```diff
@@ -237,6 +237,7 @@ impl FileCacheEntry {
                 blob_file_path,
                 blob_info.clone(),
                 Some(rafs_blob_reader),
+                None,
```

Comment:
Why pass `None` here?

Reply:
Seems fscache needs to download blob metas synchronously

Reply:
Ok, it seems implemented in `init_blobcache`, maybe we can also migrate to async runtime later.